### PR TITLE
Form validation

### PR DESCRIPTION
**Pull request by blase147**
Form validation was added to the form inputs on the index.html file.
Style.css was equally updated

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -124,87 +124,85 @@ <h1>Professional Art Printing Data</h1>
...
+  </section>
+  <footer>
+    <hr>
+    <ul class="social-icons">
       <li><a href="https://github.com/blase147"><img src="img/Icon-GitHub2.png" alt="icon"></a></li>
       <li><a href="www.linkedin.com/in/chukwuma-mosanya-346453"><img src="img/Icon-Linkedin.png" alt="icon"></a></li>
       <li><a href="www.angelist.com"><img src="img/Icon-Angelist2.png" alt="icon"></a></li>
       <li><a href="https://twitter.com/ChukwumaMosanya"><img src="img/Icon-Twitter2.png" alt="icon"></a></li>
       <li><a href="www.medium.com"><img src="img/Icon-Medium2.png" alt="icon"></a></li>
     </ul>
```
Comment (HAMMAS-SALEEM) [inline index.html:202]: ### Your Icons Container
your-icons

### Figma Icons Container
Figma-icons (copy)

- [ ] Please make all the changes shown in the Figma image above to make your icons container look similar to the Figma template :blush:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @blase147 :wave:

## Highlights :1st_place_medal:

:heavy_check_mark: Following GitHub Flow
:heavy_check_mark: Following requirements
:heavy_check_mark: Added Formspree
:heavy_check_mark: Added form validations

Good job on implementing some of the requirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_See Below Comments_

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -7,6 +11,13 @@ body {
   margin: auto;
 }
 
+@media only screen and (min-width: 1440px) {
+  .body {
+    width: 1440px;
```
Comment (ShahierNashaat) [inline style.css:16]: - [ ] [OPTIONAL] Kindly remove the fixed width as it makes the page not responsive.

File: style.css [hunk 3]
```diff
@@ -57,6 +67,20 @@ nav {
...
+@media only screen and (min-width: 1440px) {
+  .main-mobile {
+    width: 1440px;
+    height: 870px;
+  }
+}
+
+@media only screen and (min-width: 1440px) {
+  .app-bar {
+    width: 1440px;
```
Comment (ShahierNashaat) [inline style.css:79]: - [ ] [OPTIONAL] Also here remove the fixed width.

File: index.html [hunk 4]
```diff
@@ -123,89 +131,86 @@ <h1>Professional Art Printing Data</h1>
...
+        so if you'd like to chat please get in touch.</p>
+  
+      <!-- modify this form HTML and place wherever you want your form -->
+      <form class="get-in-touch" action="https://formspree.io/f/xqkjanrn" method="POST">
+          <input type="text" name="fullname" maxlength="30" placeholder="Fullname" required>
+          <input type="email" name="email" placeholder="Email address" required>
+          <textarea name="comment" cols="30" rows="10" maxlength="500" placeholder="Enter text here" required></textarea>
+          <!-- your other form fields go here -->
+          <button class="button" type="submit">Get in touch</button>
+      </form>
```
Comment (ShahierNashaat) [inline index.html:202]: - [ ] Please as the previous code reviewer mentioned you should add paddings like the Figma design.

Your Design | Figma Design
------------- | ---------------
 |

**Review by ShahierNashaat — CHANGES_REQUESTED** [review]
Hi @blase147, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 

- Correct use of GitFlow ✔️
- Descriptive commit messages ✔️
- Professional looking pull request ✔️
- Your readme looks professional ✔️
- Your code is clean and well documented ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @blase147 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

✔️ Following GitHub Flow
✔️ Added Formspree
✔️ Added form validations

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

 - [OPTIONAL] Try importing the font family used in the Figma template so your font can match and your app will look way more similar to the template.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**